### PR TITLE
COOK-004A Implemented feature for inline editing recipe names in recipe overview - also implemented a test [KS, PS, ZD]

### PR DESCRIPTION
**Pull request by pucksME**
COOK_004A Implemented feature for inline editing recipe names in recipe overview - also implemented a test [KS, PS, ZD]

Co-authored-by: Stefan Kanduth st.kanduth@gmail.com
Co-authored-by: Daniel Zirat zirat@student.tugraz.at

### CODE REVIEW

File: client/src/modules/recipes/recipesReducer.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {
     SHOW_ALL,
     ADD_RECIPE,
     ADD_RECIPE_SUCCESS,
-    ADD_RECIPE_FAILURE
+    ADD_RECIPE_FAILURE, EDIT_RECIPE_NAME, EDIT_RECIPE_NAME_SUCCESS, EDIT_RECIPE_NAME_FAILURE
```
Comment (prdngr) [inline client/src/modules/recipes/recipesReducer.js]:
```suggestion
    ADD_RECIPE_FAILURE,
    EDIT_RECIPE_NAME,
    EDIT_RECIPE_NAME_SUCCESS,
    EDIT_RECIPE_NAME_FAILURE
```

File: client/src/modules/recipesOverview/components/overview.test.js [hunk 2]
```diff
@@ -41,14 +41,18 @@ const tableContainerJson = render.create(<RecipesList recipes={generateRecipes()
...
 });
 
-test ('check if there is the correct amount of recipes inside the recipe overview table', () => {
+test('check if there is the correct amount of recipes inside the recipe overview table', () => {
     expect(tableContainerJson.children[0].children[1].children.length).toBe(10);
+});
+
+test('check if rename button exists', () => {
+    tableContainerJson.children[0].children[1].children.map(row => expect(row.children[0].children[0].children[2].type).toBe('button'));
 });
```
Comment (prdngr) [inline client/src/modules/recipesOverview/components/overview.test.js:58]:
```suggestion
test('check if rename button exists', () => {
    tableContainerJson.children[0].children[1].children.map(
        row => expect(row.children[0].children[0].children[2].type).toBe('button')
    );
});
```

**Review by prdngr — CHANGES_REQUESTED** [review]

**Review by tobiasrieger — APPROVED** [review]
Thanks for removing the package-lock.json. Looking great!

**Review by danielzirat — APPROVED** [review]
Looks Great!

**Review by prdngr — APPROVED** [review]

**Review by danielzirat — APPROVED** [review]

**Pull request merged**